haystack/components/generators/chat/hugging_face_api.py

```diff
@@ -272,9 +275,14 @@ def _run_streaming(self, messages: List[Dict[str, str]], generation_kwargs: Dict
 
             finish_reason = choice.finish_reason
 
-            meta = {}
-            if finish_reason:
-                meta["finish_reason"] = finish_reason
+            meta = {
+                "finish_reason": finish_reason if finish_reason else None,
+                "received_at": datetime.now().isoformat(),
+            }
+            chunks_meta.append(meta)
+
+            if first_chunk_time is None:
+                first_chunk_time = meta["received_at"]
```

Comment:
```suggestion
            meta = {}
            if finish_reason:
                meta["finish_reason"] = finish_reason

            if first_chunk_time is None:
                first_chunk_time = datetime.now().isoformat()
```

Do you see any problems with this simpler approach?